Doc/library/dis.rst

```diff
@@ -851,13 +851,13 @@ iterations of the loop.
 .. opcode:: STORE_NAME (namei)
 
    Implements ``name = STACK.pop()``. *namei* is the index of *name* in the attribute
-   :attr:`co_names` of the code object. The compiler tries to use
+   :attr:`!codeobject.co_names` of the code object. The compiler tries to use
    :opcode:`STORE_FAST` or :opcode:`STORE_GLOBAL` if possible.
 
 
 .. opcode:: DELETE_NAME (namei)
 
-   Implements ``del name``, where *namei* is the index into :attr:`co_names`
+   Implements ``del name``, where *namei* is the index into :attr:`!codeobject.co_names`
```

Comment:
I think we should prefer not prefixing with ``codeobject`` -- currently when read out loud the markup makes sense, but with this change we are saying ``namei`` is the index into the ``codeobject.co_names`` attribute of the code object, which doesn't make sense.
```suggestion
   Implements ``del name``, where *namei* is the index into :attr:`!co_names`
```

Reply:
I agree, this displays as `codeobject.co_names` which looks odd.